solr/core/src/java/org/apache/solr/metrics/AggregateMetric.java

```diff
@@ -93,16 +99,13 @@ public double getMax() {
     if (values.isEmpty()) {
       return 0;
     }
-    Double res = null;
+    double res = 0;
     for (Update u : values.values()) {
       if (!(u.value instanceof Number)) {
+        log.warn("not a Number: " + u.value);
```

Comment:
Note I'm not completely clear whether `u.value` is ever _expected_ to not be a `Number` - have seen this line report `false` and `LocalStatsCache` and I'm tracing back through to find out why these occur.